src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.test.js

```diff
@@ -289,79 +289,75 @@ describe('buildAvEmbedURL', () => {
     it('should build an embed URL for the page', () => {
       expect(
         getEmbedUrl({
-          mediaId: '/serbian/cyr/av-embeds/srbija-68707945',
+          mediaId: 'serbian/cyr/srbija-68707945',
           type: 'avEmbed',
           queryString: '',
         }),
-      ).toEqual('https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945');
+      ).toEqual('/ws/av-embeds/cps/serbian/cyr/srbija-68707945');
     });
 
     it('should build an embed with a vpid URL for the page', () => {
       expect(
         getEmbedUrl({
-          mediaId: '/serbian/cyr/av-embeds/srbija-68707945/vpid/p0cfmdwj',
+          mediaId: 'serbian/cyr/srbija-68707945/vpid/p0cfmdwj',
           type: 'avEmbed',
           queryString: '',
         }),
-      ).toEqual(
-        'https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945/vpid/p0cfmdwj',
-      );
+      ).toEqual('/ws/av-embeds/cps/serbian/cyr/srbija-68707945/vpid/p0cfmdwj');
     });
 
     it('should build an embed with a pid URL for the page', () => {
       expect(
         getEmbedUrl({
-          mediaId: '/serbian/cyr/av-embeds/srbija-68707945/pid/p0cfmdwn',
+          mediaId: 'serbian/cyr/srbija-68707945/pid/p0cfmdwn',
           type: 'avEmbed',
           queryString: '',
         }),
-      ).toEqual(
-        'https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945/pid/p0cfmdwn',
-      );
+      ).toEqual('/ws/av-embeds/cps/serbian/cyr/srbija-68707945/pid/p0cfmdwn');
     });
   });
 
   describe('service without variant', () => {
     it('should build a media embed URL', () => {
       expect(
         getEmbedUrl({
-          mediaId: '/russian/av-embeds/media-38886884',
+          mediaId: 'russian/media-38886884',
           type: 'avEmbed',
           queryString: '',
         }),
-      ).toEqual('https://www.bbc.com/russian/av-embeds/media-38886884');
+      ).toEqual('/ws/av-embeds/cps/russian/media-38886884');
     });
 
     it('should build a media embed URL omitting renderer env', () => {
       expect(
         getEmbedUrl({
-          mediaId: '/russian/av-embeds/features-49881797?renderer_env=live',
+          mediaId: 'russian/features-49881797?renderer_env=live',
           type: 'avEmbed',
           queryString: '',
         }),
-      ).toEqual('https://www.bbc.com/russian/av-embeds/features-49881797');
+      ).toEqual(
+        '/ws/av-embeds/cps/russian/features-49881797?renderer_env=live',
```

Comment:
The expectation no longer matches the the test description... should renderer_env be included in the URL or not?